### PR TITLE
Document token acquirement and authentication

### PR DESCRIPTION
**Pull request by timkurvers**
More documentation soon™

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,10 +102,56 @@ Alternatively, these environment variables will be picked up automatically:
 
 ### Token acquirement
 
-Soon.
```
Comment (sindrenm) [inline README.md]: Good.

File: README.md [hunk 2]
```diff
@@ -102,10 +102,55 @@ Alternatively, these environment variables will be picked up automatically:
...
+to know what to do with any newly obtained tokens. To boot back into a native
+applicaton one could use custom URL schemes or intents:
+
+```ruby
+OpenIDTokenProxy.configure do |config|
+  config.token_acquirement_hook = proc { |token|
+    "my-app://?token=#{token}&refresh_token=#{token.refresh_token}"
+  }
+end
+```
```
Comment (sindrenm) [inline README.md]: Awesome!

File: README.md [hunk 3]
```diff
@@ -102,10 +102,55 @@ Alternatively, these environment variables will be picked up automatically:
...
+
+  ...
+end
+```
+
+Access tokens may be provided as follows:
+
+- `X-Token` header.
+- `Authorization: Bearer <token>` header.
+- Query string parameter `token`.
```
Comment (sindrenm) [inline README.md]: You only need one of these, right? Cause I read it like all three are required, which seems strange.

Comment (timkurvers) [inline README.md]: Ah, good point. Any proposals for rephrasing?

Comment (sindrenm) [inline README.md]: Something like “Access token may be provided with either of the following:” perhaps?

**Comment by sindrenm** [comment]
Great stuff. :fish_cake:

**Comment by timkurvers** [comment]
:facepunch:

**Pull request merged**